public/sass/CoursePlanner.scss

```diff
@@ -102,36 +95,71 @@ body {
 }
 
 .add-class-btn {
+    padding: 5px !important;
     margin-top: 5px;
+    margin-bottom: 50px;
     //courseCol width - 15
-    width: 73px;
+    width: 75px;
```

Comment:
Is it worth making the courseCol width a variable so you can actually do $courseColWidth - 15px here using sass?